hack/e2e-pool-test.sh

```diff
@@ -242,21 +244,34 @@ verify_pool_cd_imagesets $FAKE_POOL_NAME $IMAGESET_NAME
 create_imageset fake-cis
 # Modify the clusterpool and watch CDs get replaced
 oc patch clusterpool $FAKE_POOL_NAME --type=merge -p '{"spec":{"imageSetRef": {"name": "fake-cis"}}}'
-expect_all_clusters_current $FAKE_POOL_NAME False
+oc wait --for=condition=AllClustersCurrent=false --timeout=1m clusterpool/$FAKE_POOL_NAME
```

Comment:
Is it worth replacing the other call to `expect_all_clusters_current` with `oc wait`?

Reply:
Some of them maybe, but definitely not universally. The `oc wait` is for when we’re expecting a state transition, whereas the other is where we’re expecting the pool to already be in a particular state. Could be a separate effort to go through and audit each occurrence.